dev

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+main() {
+    case $1 in
+        up)
+            build
+            ;;
+        down)
+            docker-compose down
+            ;;
+        shell)
+            docker-compose exec wordpress bash
+            ;;
+        test)
+            if [ ! $1 ]; then
+                set -- "--help"
+            fi
+            docker-compose exec wordpress codecept.sh run --env docker "${@:2}"
+            ;;
+        debug)
+            shift
+            if [ ! "$1" ]; then
+                echo "usage $0 debug <on|off>"
+                exit 1
+            fi
+            HOST=$(ifconfig | grep 'inet ' | grep -v '127.0.0.1' | head -1 | awk '{print $2}')
+            docker-compose exec wordpress xdebug.sh $1 $HOST 9000
+            docker-compose restart wordpress
+            ;;
+        *)
+            cat <<EOF
+Development Environment Manager
+
+Usage: $0 <command> [options]
+
+The following commands are supported:
+
+up       Start development environment (builds if necessary)
+down     Destroy development environment (also clears the DB)
+shell    Enter wordpress container shell
+test     Run Codeception tests
+debug    Turn debugging (xdebug) on or off
+EOF
+            exit 1
+            ;;
+    esac
+}
+
+# run a wp-cli command
+wp() {
+    docker-compose exec wordpress wp --allow-root "$@"
```

Comment:
this should probably be `docker-compose exec --user=noroot wordpress wp "$@"` following the new convention of using a non-root user to run these things, it tends to cause less problems with permissions